src/category_theory/monoidal/category.lean

```diff
@@ -316,7 +344,27 @@ by rw [←tensor_left_iff, ←cancel_epi (α_ (𝟙_ C) (𝟙_ _) (𝟙_ _)).hom
        triangle, ←right_unitor_tensor, right_unitor_naturality]
 
 lemma unitors_inv_equal : (λ_ (𝟙_ C)).inv = (ρ_ (𝟙_ C)).inv :=
-by { ext, simp [←unitors_equal], }
+by { ext, simp [←unitors_equal] }
+
+@[reassoc]
+lemma right_unitor_inv_comp_tensor (f : W ⟶ X) (g : 𝟙_ C ⟶ Z) :
+  (ρ_ _).inv ≫ (f ⊗ g) = f ≫ (ρ_ _).inv ≫ (𝟙 _ ⊗ g) :=
+by {slice_rhs 1 2 { rw right_unitor_inv_naturality }, simp }
+
+@[reassoc]
+lemma right_unitor_inv_sliding_right (f : W ⟶ X) (g : 𝟙_ C ⟶ Z) :
+  f ≫ (ρ_ _).inv ≫ (𝟙 _ ⊗ g) = (ρ_ _).inv ≫ (𝟙 _ ⊗ g) ≫ (f ⊗ 𝟙 _) :=
+by { rw ←right_unitor_inv_comp_tensor, simp }
```

Comment:
Do you actually need this lemma, or the other `sliding` lemma below? I can't see them used elsewhere, and I think they're getting a bit too baroque.

Reply:
I'll delete them for now, maybe we can add a bunch of more general sliding lemmas at some point.